packages/fxa-admin-server/scripts/db-helpers.ts

```diff
@@ -0,0 +1,126 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { uuidTransformer } from 'fxa-shared/db/transformers';
+import {
+  Device,
+  Email,
+  Account,
+  SessionToken,
+  SignInCodes,
+} from 'fxa-shared/db/models/auth';
+import crypto from 'crypto';
+import { setupAuthDatabase } from 'fxa-shared/db';
+import { ILogger } from 'fxa-shared/log';
+
+const toRandomBuff = (size: number) =>
+  uuidTransformer.to(crypto.randomBytes(size).toString('hex'));
+
+export const toZeroBuff = (size: number) =>
+  Buffer.from(Array(size).fill(0)).toString('hex');
```

Comment:
I might be confused by these two functions (`toRandomBuff()` and `toZeroBuff()`).
If I understand correctly `toRandomBuff()` will return a Buffer, but `toZeroBuff()` might be returning a string?
If so, is the function name misleading, or should we just return the raw buffer and then call `.toString('hex')` separately (like on L28-29 below?

Not sure if these need explicit return types either. I'm sure TypeScript is great at inferring types, but explicit ":string" or ":buffer" might be useful if they are indeed different return types.